src/components/TodoApp/TodoApp.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import { TodoList } from '../TodoList';
+import { TodosFilter } from '../TodosFilter';
+import { NewTodo } from '../NewTodo';
+import { TodosToggler } from '../TodosToggler';
+import { TodoCount } from '../TodoCount';
+
+export const TodoApp = () => {
+  const [todos, setTodos] = useState([]);
+  const [toggleAll, setToggleAll] = useState(false);
+  const [filterValue, setFilterValue] = useState('All');
```

Comment:
comment is not fixed, create a constant object FILTER with corresponding fields

Reply:
OK